src/frontend/src/hooks/recruitment.hooks.ts

```diff
@@ -21,6 +30,38 @@ export const useAllMilestones = () => {
   });
 };
 
+export const useDeleteMilestone = () => {
+  const queryClient = useQueryClient();
+  return useMutation<{ message: string }, Error, any>(
+    ['milestones', 'delete'],
+    async (milestoneId: string) => {
+      const { data } = await deleteMilestone(milestoneId);
+      return data;
+    },
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries(['milestones']);
+      }
+    }
+  );
+};
+
+export const useDeleteFAQ = () => {
+  const queryClient = useQueryClient();
+  return useMutation<{ message: string }, Error, any>(
+    ['faqs', 'delete'],
+    async (faqId: string) => {
+      const { data } = await deleteFaq(faqId);
+      return data;
+    },
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries(['faqs']);
+      }
+    }
+  );
+};
+
```

Comment:
can you move this under the faqs stuff and more the milestone one under the milestone one just for organization